### ISSUE TITLE
Budget analysis / projection to understand the resources we have

### ISSUE THREAD

**Issue by choldgraf** [2i2c-org/team-compass#296]
### Description

In a recent team meeting we discussed that some parts of 2i2c's work is being strained due to not having enough capacity. This included engineering and technical things, as well as more "meta" things like administration, project management, etc.

If we wish to grow our capacity, we need to understand our budget situation first. Now that we have settled with CS&S, we should be able to understand our budget picture and make some projections. This issue will track that effort.

### Tasks to complete

- [x] Write down our current funds available and burn rate
- [x] Make a few budget projections under
  - different scenarios of income
  - different scenarios of expenses (e.g. hiring different kinds of roles)
- [x] Run this budget by the CS&S admin team to get feedback
  - [x] Clean up the budget and separate out the budget items in CS&S vs. in NumFocus
  - [x] Send CS&S team an update to describe the high-level view
- [x] Decide how we'd like to encode this budget information in our documentation and next steps

**Comment by choldgraf** [comment]
### Update: a lightweight model

I've put together a lightweight model using a little web app called startuprunway.io . You'll need credentials to access it, so if you're interested, please ping me and I will send them to you.

**To access**

- Ask me for access (I can share via bitwarden)
- Go to https://startuprunway.io/scenarios/46543
- It will send you to a sign in window
- Input the username/password that I've shared with you
- In the top left, there's a dropdown. Select "2i2c Bootstrap"
- You'll be taken to a screen with a plot of our burn rate. It'll look like this:

![image](https://user-images.githubusercontent.com/1839645/143148587-c3357bd4-e8b1-4b29-9bb9-faadc657e638.png)

**Comment by yuvipanda** [comment]
This is super helpful,, @choldgraf

**Comment by colliand** [comment]
I agree! Thanks Chris. It will be interesting to model some scenarios arising from revenue through sale of managed hub services. That work and related discussions within our team should help us identify:

+ sales revenue targets to achieve sustainability
+ gaps in our capacity to hit those targets
+ possible need for new fundraising

**Comment by choldgraf** [comment]
### Update

I've got a meeting with Joe and Page at CS&S to go over these budget projections and decide whether they look correct and what kind of funding situation we have for hiring somebody new

**Comment by choldgraf** [comment]
### Update: Meeting with CS&S

We had a meeting with Keith, Page, and Joe at CS&S to discuss financial planning and the impact that hiring would have.

tl;dr: we have enough money to hire another team member, and we had some good discussions about how to plan for future hires etc (see below).

Here are a few major takeaways:

#### A common financial plan organizations follow

- Keith recommended that we shoot for a minimum of 6 months of cash on-hand (this is more than your typical non-profit)
- We agreed it was reasonable to boost this to 12 months, with the following general breakdown:
  - **9 months** are a funding buffer**. If we start to dip into this, it's time to prioritize fundraising either via grants or more aggressively getting contracts.
  - **3 months are an emergency "wind-down" fund**. If we ever have to start spending this, we likely need to begin winding down 2i2c, and will use these funds use it to ensure everybody's salaries are paid for the three months as we close out operations and look for other 

#### How does this relate to our funding?

A few assumptions:

- As of September 2021, we have about $1.5m in expected funding.
- To simplify things in the projection below, we'll treat that funding as if it were all there at once.
- Assume we have **no hub revenue** in that time, to look at our burn-down **purely on grants**.
- Assume that salaries don't change over time, and are restricted to our current paid staff + potential roles we can toggle to look at their impact.
- Assume we don't want to run out of funding before November 2023, because that is the end-date of our CZI grant, and it would look bad if we ran out of funding *before* we got to that date :-)

Here's what our funding situation looks like w/ the above assumptions:

![image](https://user-images.githubusercontent.com/1839645/151595308-709ac50f-ee0a-4a35-9786-2edd67c29cc8.png)
  
We'd run out of funding in May 2024, and so our 12 month "warning checkpoint" would be May 2023.

If we assume that a community success lead is hired at around $160K total a year, it would have the following impact:

![image](https://user-images.githubusercontent.com/1839645/151595483-defac91f-03c4-4837-9d8d-2b574041c2cd.png)

We'd run out of funding in December 2023, and so our 12 month "warning checkpoint" would be December 2022.

#### Conclusion: we can hire somebody

We conclude that if we hired another team member then we would remain within the funding runway that we want to keep on-hand.

**Comment by damianavila** [comment]
Interesting analysis, thanks for the summary!!

**Comment by choldgraf** [comment]
I'm going to close this one since we have a path forward on the hire. It's still unclear what's the right way to integrate this kind of budgeting in with our broader financial planning, but I don't want to blog progress on that question
